INSTALL.md

```diff
@@ -1088,7 +1088,7 @@ This is only supported on systems where loading of shared libraries is supported
 
 Enables qlog output support for the QUIC protocol. This functionality is
 unstable and implements a draft version of the qlog specification. The qlog
-output from OpenSSL will change in incompatible ways in future, and is not
+output from OpenSSL will change in incompatible ways in the future and is not
```

Comment:
There is definitely a pause after `future` when reading this sentence, so I think the comma should stay: it's not a list, but separating phrases

Reply:
I see your point about it not being a list, however, I believe a comma should only be used before "and" if it is connecting two independent clauses. In this instance, the output remains the subject of the sentence, making it dependent. A good example of this is described here: https://www.scribbr.com/commas/comma-before-or-after-and/

Reply:
I'm not a fan of prescriptive grammar, and OpenSSL doesn't have an English language style guide (beyond asking for British English, it defers to the LDP, which explicitly states that it does not have one). We generally change prose if it's clearly wrong, but there have been plenty of PRs that start by fixing obvious issues but keep going with less and less obvious changes until they are arguing about matters of preference and then don't get merged.

In lieu of a style guide (our main deliverable is C code, and we do have a style guide for that) the general guide seems to be "leave alone if it's not clearly wrong".

Here, the additional `the` is clearly necessary, but removing the comma makes a long sentence that needs breaking up